{"uid":"f110fa18eb286c0d","name":"test_Login_Login_with_Google","fullName":"tests.test_suite_demo.LoginTest#test_Login_Login_with_Google","historyId":"0c43215cb02488ae8f5707a0aac395ed","time":{"start":1660121426913,"stop":1660121546697,"duration":119784},"status":"failed","statusMessage":"AssertionError: Login with Google failed","statusTrace":"self = <tests.test_login.LoginTest testMethod=test_Login_Login_with_Google>\n\n    @allure.story('KeyCloak')  # epic/story of the test case\n    # @allure.severity(allure.severity_level.MINOR) # severity of the test case\n    # @pytestrail.case('C48') # test case if on TestRail\n    def test_Login_Login_with_Google(self):\n        self.homeNavigation = HomeNavigation(self.driver)\n        self.loginPage = LoginPage(self.driver)\n        self.loggedInPage = LoggedInPage(self.driver)\n        self.ts = TestStatus(self.driver)\n    \n        with allure.step('Waiting for homepage to completely load'):\n            self.ts.markFinal(self.page_has_loaded(), \"Homepage loaded completely\", \"Homepage not loaded completely\")\n    \n        with allure.step('Navigate to login page'):\n            self.homeNavigation.goToLoginPage()\n            self.ts.markFinal(self.loginPage.isAt, \"Navigation to login page\", \"Navigation to login page failed\")\n    \n        with allure.step('Login with Google credentials'):\n            self.loginPage.loginWithGoogle(username=td.testData(\"loginData.google.username\"), password=td.testData(\"loginData.google.password\"))\n>           self.ts.markFinal(self.loggedInPage.isAt, \"Login with Google successful\", \"Login with Google failed\")\n\ntests/test_login.py:179: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <utilities.teststatus.TestStatus object at 0x7fb58eec17e0>\nresult = False, passResultMessage = 'Login with Google successful'\nfailResultMessage = 'Login with Google failed'\n\n    def markFinal(self, result, passResultMessage, failResultMessage):\n        \"\"\"\n        Mark the final result of the verification point in a test case\n        This needs to be called at least once in a test case\n        This should be final test status of the test case\n        \"\"\"\n        self.setResult(result, passResultMessage, failResultMessage)\n        if \"FAIL\" in self.resultList:\n            self.resultList.clear()\n>           assert False, failResultMessage\nE           AssertionError: Login with Google failed\n\nutilities/teststatus.py:55: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setUp","time":{"start":1660121408989,"stop":1660121426911,"duration":17922},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"_unittest_setUpClass_fixture_LoginTest","time":{"start":1660121408989,"stop":1660121408989,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"oneTimeSetUp","time":{"start":1660121408989,"stop":1660121408989,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Login with Google failed","statusTrace":"self = <tests.test_login.LoginTest testMethod=test_Login_Login_with_Google>\n\n    @allure.story('KeyCloak')  # epic/story of the test case\n    # @allure.severity(allure.severity_level.MINOR) # severity of the test case\n    # @pytestrail.case('C48') # test case if on TestRail\n    def test_Login_Login_with_Google(self):\n        self.homeNavigation = HomeNavigation(self.driver)\n        self.loginPage = LoginPage(self.driver)\n        self.loggedInPage = LoggedInPage(self.driver)\n        self.ts = TestStatus(self.driver)\n    \n        with allure.step('Waiting for homepage to completely load'):\n            self.ts.markFinal(self.page_has_loaded(), \"Homepage loaded completely\", \"Homepage not loaded completely\")\n    \n        with allure.step('Navigate to login page'):\n            self.homeNavigation.goToLoginPage()\n            self.ts.markFinal(self.loginPage.isAt, \"Navigation to login page\", \"Navigation to login page failed\")\n    \n        with allure.step('Login with Google credentials'):\n            self.loginPage.loginWithGoogle(username=td.testData(\"loginData.google.username\"), password=td.testData(\"loginData.google.password\"))\n>           self.ts.markFinal(self.loggedInPage.isAt, \"Login with Google successful\", \"Login with Google failed\")\n\ntests/test_login.py:179: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <utilities.teststatus.TestStatus object at 0x7fb58eec17e0>\nresult = False, passResultMessage = 'Login with Google successful'\nfailResultMessage = 'Login with Google failed'\n\n    def markFinal(self, result, passResultMessage, failResultMessage):\n        \"\"\"\n        Mark the final result of the verification point in a test case\n        This needs to be called at least once in a test case\n        This should be final test status of the test case\n        \"\"\"\n        self.setResult(result, passResultMessage, failResultMessage)\n        if \"FAIL\" in self.resultList:\n            self.resultList.clear()\n>           assert False, failResultMessage\nE           AssertionError: Login with Google failed\n\nutilities/teststatus.py:55: AssertionError","steps":[{"name":"Waiting for homepage to completely load","time":{"start":1660121426914,"stop":1660121435706,"duration":8792},"status":"passed","steps":[],"attachments":[{"uid":"5294765b3107ce87","name":"Homepage loaded completely.1660121428846.png","source":"5294765b3107ce87.png","type":"image/png","size":52402}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},{"name":"Navigate to login page","time":{"start":1660121435706,"stop":1660121463694,"duration":27988},"status":"passed","steps":[],"attachments":[{"uid":"951847601a30334f","name":"Navigation to login page.1660121445109.png","source":"951847601a30334f.png","type":"image/png","size":164992}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},{"name":"Login with Google credentials","time":{"start":1660121463703,"stop":1660121546615,"duration":82912},"status":"failed","statusMessage":"AssertionError: Login with Google failed\n","statusTrace":"  File \"/home/runner/work/FlyerrUIAutomation/FlyerrUIAutomation/tests/test_login.py\", line 179, in test_Login_Login_with_Google\n    self.ts.markFinal(self.loggedInPage.isAt, \"Login with Google successful\", \"Login with Google failed\")\n  File \"/home/runner/work/FlyerrUIAutomation/FlyerrUIAutomation/utilities/teststatus.py\", line 55, in markFinal\n    assert False, failResultMessage\n","steps":[],"attachments":[{"uid":"b52622fb6ef54ed","name":"Login with Google failed.1660121546245.png","source":"b52622fb6ef54ed.png","type":"image/png","size":52749}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[{"uid":"16deab54a2251498","name":"log","source":"16deab54a2251498.txt","type":"text/plain","size":2708},{"uid":"54ebfef68b1e5570","name":"stdout","source":"54ebfef68b1e5570.txt","type":"text/plain","size":100},{"uid":"1afdc95b06bf378a","name":"stderr","source":"1afdc95b06bf378a.txt","type":"text/plain","size":26772}],"parameters":[],"stepsCount":3,"attachmentsCount":6,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"setUp::0","time":{"start":1660121547011,"stop":1660121547113,"duration":102},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"oneTimeSetUp::0","time":{"start":1660121547131,"stop":1660121547131,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"_unittest_setUpClass_fixture_LoginTest::0","time":{"start":1660121547132,"stop":1660121547133,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"story","value":"KeyCloak"},{"name":"tag","value":"@pytest.mark.usefixtures('oneTimeSetUp', 'setUp')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_suite_demo"},{"name":"subSuite","value":"LoginTest"},{"name":"host","value":"fv-az177-801"},{"name":"thread","value":"2355-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_suite_demo"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"f81808d5d8170398","reportUrl":"https://suresh-chunduru.github.io/FlyerrUIAutomation/20//#testresult/f81808d5d8170398","status":"failed","statusDetails":"AssertionError: Login with Google failed","time":{"start":1660120885868,"stop":1660121004773,"duration":118905}}]},"tags":["@pytest.mark.usefixtures('oneTimeSetUp', 'setUp')"]},"source":"f110fa18eb286c0d.json","parameterValues":[]}